enhancements/windows-containers/container-runtime-containerd.md

```diff
@@ -0,0 +1,247 @@
+---
+title: container-runtime-containerd
+authors:
+  - "@selansen"
+reviewers:
+  - "@openshift/openshift-team-windows-containers"
+    approvers:
+  - "@aravindhp"
+  - "@mrunalp"
+creation-date: 2021-11-19
+last-updated: 2021-12-05
+status: implementable
+---
+
+# containerd - Windows container runtime
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [x] Design details are appropriately documented from clear requirements
+- [x] Test plan is defined
+- [x] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [OpenShift-docs](https://github.com/OpenShift/OpenShift-docs/)
+
+## Summary
+
+This enhancement intends to allow customers to bring up Windows nodes with containerd
+as the default runtime from OpenShift 4.11 onwards. When customers upgrade clusters to
+OpenShift 4.11, container runtime will be migrated from Docker to containerd
+
+## Motivation
+
+In Kubernetes, the container runtime interface has been used to talk to a container runtime. From
```

Comment:
the kubelet talks to a container runtime (like CRI-O or containerd) using the CRI (Container Runtime Interface)